docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.9"
+services:
+  api:
+    build:
+      context: ./api
+      dockerfile: Dockerfile.dev
+    volumes:
+      - ./api:/home/app/api
+    ports:
+      - 8000:8000
+    command: python manage.py runserver 0.0.0.0:8000
```

Comment:
We have a similar command in the `Dockerfile.dev`, what's the purpose of this here?

Reply:
This is the command for docker-compose only, in future if we want to change the CMD in Dockerfile for running only if a single container started without docker-compose, than this command will overwrite Dockerfile's command and used only in docker-compose.